test/unit/db/seeds_test.rb

```diff
@@ -114,6 +114,7 @@ def teardown
     assert tenant_account.approved?
     assert tenant_account.state_changed_at.present?
     assert_equal ApplicationPlan.find_by(name: 'enterprise').id, tenant_account.bought_cinstance&.plan.id
+    assert_equal 'API', tenant_account.default_service&.name
```

Comment:
This is weird ...
Why do we check for nil in a test?

Reply:
just to don't raise a weird error if it is not there. It is a clearer message imo. Anyway, small improvements tomorrow instead of today plz :joy:

Reply:
But why in a test????
So the test will fail anyway, `'API' == nil` or error raising makes the test fail... but does not have the same meaning.

When reading it, one can get confused that it could be absent and thus "fix" it with `assert_nil`

Reply:
Then we could do:
```ruby
assert (service = tenant_account.default_service)
assert_equal 'API', service.name
```